### DISCUSSION TITLE
Upgrading the plugins packages to be compatible with the new Grapesjs Version

### DISCUSSION THREAD

**Discussion by kanaihyakumar** [GrapesJS/grapesjs#5410] [Ideas]
There are two packages which still outdated due to which there is an issue. It will be really great if anyone upgrade the below plugins to be compatible with the newer version:

1. [parser-postcss](https://github.com/GrapesJS/parser-postcss)
2. [style-gradient](https://github.com/GrapesJS/style-gradient)

Thank you in advance

**Comment by artf** [comment]
@kanaihyakumar I'll be happy to merge PRs with the updates

**Reply by kanaihyakumar** [reply]
Thank you for the response. I have raised a pull request with changes it needed. If u merge it will be great.
